edge-util/src/Microsoft.Azure.Devices.Edge.Util/TwinHeader.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class TwinHeader : IEquatable<TwinHeader>
+    {
+        [JsonConstructor]
+        public TwinHeader(string cert1, string cert2)
+        {
+            this.Cert1 = cert1;
+            this.Cert2 = cert2;
+        }
+
+        [JsonProperty("cert1")]
+        public string Cert1 { get; }
```

Comment:
This is to just split the cert into 2 parts right? Can we use an array instead?

Reply:
@varunpuranik this is going in the module Twin... Just confirming you want to use the relatively new feature of allowing arrays in the Twin?